apis/models/userDao.js

```diff
@@ -1,18 +1,29 @@
 const database = require('./dataSource');
 
-// const signUp = async () => {
-//     await database.query(
-
-//     )
-// }
-
-// const signIn = async () => {
-//     await database.query(
-
-//     )
-// }
+const getUserByUsername = async (username) => {
+  const [user] = await database.query(
+    `SELECT
+            id,
+            username,
+            fullname,
+            password,
+            phone_number,
+            address,
+            gender,
+            birth
+    FROM users
```

Comment:
sql문도 줄이 바뀌면 2칸으로 들여쓰기 해주시는 것이 가독성에 더 좋을 것 같습니다.
```suggestion
  const [user] = await database.query(
    `SELECT
        id,
        username,
        fullname,
        password,
        phone_number,
        address,
        gender,
        birth
    FROM users
```

Reply:
수정